src/sfctl/tests/chaos_test.py

```diff
@@ -0,0 +1,68 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom Chaos command related tests"""
+import unittest
+import sfctl.custom_chaos as sf_c
+
+class ChaosTests(unittest.TestCase):
+    """Chaos tests"""
+
+    def parse_none_context_test(self):
+        """Parsing None context returns None"""
+        res = sf_c.parse_chaos_context(None)
+        self.assertIs(res, None)
+
+    def parse_populated_context_test(self):
+        """Parse context with contents"""
+        from azure.servicefabric.models.chaos_context import (
+            ChaosContext
+        )
+
+        wrapper = sf_c.parse_chaos_context({
+            'key1': 'value1',
+            'key2': 'value2',
+            'key3': 'value3'
+        })
+
+        self.assertIsInstance(wrapper, ChaosContext)
+        res = wrapper.map
+        self.assertIsInstance(res, dict)
+        self.assertEqual(len(res), 3)
+        self.assertEqual(res['key1'], 'value1')
+        self.assertEqual(res['key2'], 'value2')
+        self.assertEqual(res['key3'], 'value3')
+
+    def parse_none_target_filter_test(self):
+        """Parse None chaos target filter returns None"""
+        self.assertIs(sf_c.parse_chaos_target_filter(None), None)
+
+    def parse_nodetype_list_test(self):
+        """Parse nodetypeinclusionlist list"""
+
```

Comment:
Don't need this newline here.

Reply:
It appears in other tests, e.g., health_test, a newline is introduced after function def and import (no newline between def and import but after that there is a newline). I thought my newline belongs to that type of newline as I do not have an import.

Reply:
tanv1r is actually me, I was signed in with a different id